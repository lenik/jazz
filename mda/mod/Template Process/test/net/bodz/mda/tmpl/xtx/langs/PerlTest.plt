Number system: 

¬ use strict; 
¬ my @num = ('zero', "one", "two", "three", 
¬       qw(four five six seven eight nine), 'ten'); 
¬ for (my $i = 0; $i < 10; $i++) {
    Number { $i } is { $num[$i] }. 
¬ }
