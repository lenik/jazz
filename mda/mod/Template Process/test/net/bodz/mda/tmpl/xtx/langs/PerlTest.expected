Number system: 

    Number 0 is zero. 
    Number 1 is one. 
    Number 2 is two. 
    Number 3 is three. 
    Number 4 is four. 
    Number 5 is five. 
    Number 6 is six. 
    Number 7 is seven. 
    Number 8 is eight. 
    Number 9 is nine. 
